packages/synapse-react-client/src/components/DataGrid/DataGrid.tsx

```diff
@@ -42,10 +43,10 @@ const DataGrid = () => {
     getModel,
   } = useDataGridWebSocket()
   useEffect(() => {
-    if (presignedUrl) {
-      createWebsocket(presignedUrl)
+    if (replicaId && presignedUrl) {
+      createWebsocket(replicaId, presignedUrl)
```

Comment:
Pass the `replicaId` from the server to the websocket class instance

Reply:
We aren't using the `replicaId` that the server gives us, and passing it to the websocket class ensures that we do.